ovh/resource_cloud_project_kube_iprestrictions_test.go

```diff
@@ -0,0 +1,127 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func init() {
```

Comment:
this sweeper is probably useless as the ovh_cloud_project_kube existing sweeper will delete all clusters with a name starting the the test prefix. this code can be safely removed

Reply:
Fixed.